### PR TITLE
remove unused constants

### PR DESCRIPTION
**Pull request by islinwb**

### CODE REVIEW

**Comment by ijsnellf** [comment]
Please fix the linter errors. Run `make lint` to test them locally.

**Comment by ozevren** [comment]
Changes look good to me, but please rebase and fix linter errors.

File: tests/e2e/tests/dashboard/dashboard_test.go [hunk 1]
```diff
@@ -41,8 +41,6 @@ import (
 
 const (
 	istioMeshDashboard = "install/kubernetes/helm/istio/charts/grafana/dashboards/istio-mesh-dashboard.json"
-	serviceDashboard   = "install/kubernetes/helm/istio/charts/grafana/dashboards/istio-service-dashboard.json"
-	workloadDashboard  = "install/kubernetes/helm/istio/charts/grafana/dashboards/istio-workload-dashboard.json"
```
Comment (baodongli) [inline tests/e2e/tests/dashboard/dashboard_test.go]: There are commented out code that refers to the above two constants.
@douglas-reid Can you please comment on this?

Comment (douglas-reid) [reply tests/e2e/tests/dashboard/dashboard_test.go]: Yes. That code needs to be revived. It was commented out to unblock a release due to changes in dashboards at the last minute. Those tests are needed and we should not remove these consts.

**Pull request closed without merging**